doc/design/diagrams/puml_github_workflow.svg

```diff
@@ -1,6 +1,6 @@
-<?xml version="1.0" encoding="UTF-8" standalone="no"?><svg xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink" contentScriptType="application/ecmascript" contentStyleType="text/css" height="405px" preserveAspectRatio="none" style="width:1007px;height:405px;background:#EEEBDC;" version="1.1" viewBox="0 0 1007 405" width="1007px" zoomAndPan="magnify"><defs><filter height="300%" id="f18ey02a78uejz" width="300%" x="-1" y="-1"><feGaussianBlur result="blurOut" stdDeviation="2.0"/><feColorMatrix in="blurOut" result="blurOut2" type="matrix" values="0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 .4 0"/><feOffset dx="4.0" dy="4.0" in="blurOut2" result="blurOut3"/><feBlend in="SourceGraphic" in2="blurOut3" mode="normal"/></filter></defs><g><!--MD5=[81da3b3e20e7b00bc69cac0acd70a916]
```

Comment:
We need to think about whether we want to actually store svg files in git. Small changes in the file used to generate it lead to large changes in the svg. I do not know whether we plan to have many svg files in the repository and if so, if they are mostly static it is less problematic from a storage point of view.

Reply:
It is recommended to use the script in the tools folder for generating the vectorgraphics: https://github.com/eclipse-iceoryx/iceoryx/blob/master/tools/generate_plantuml_svg.sh
With that, a defined version of PlantUML is used to create the svg and therefore reduce the diff in the svg files.
Personally i think that the diff in this file is not that large (only the size and some coordinates differ) and Github will be able to tackle that.

Reply:
The files are also quite small and improve the comprehension alot